packages/replayio/src/utils/recordings/upload/uploadSourceMaps.ts

```diff
@@ -1,5 +1,4 @@
 import fs from "fs/promises";
-import promiseMap from "p-map";
```

Comment:
Would be nice for us to also remove the other "p-map" reference (in `uploadRecording`) then we could remove the dependency entirely.

Reply:
that's already done in https://github.com/replayio/replay-cli/pull/368 :)